_shared/project/requirements/checkformatting.in

```diff
@@ -1,4 +1,5 @@
 pip-tools
+pip-sync-faster
```

Comment:
Oops! The cookiecutter was [still using `pip-sync` directly](https://github.com/hypothesis/cookiecutters/blob/de5ce7e813051f383e22beb84114db9cf8e3755f/_shared/project/tox.ini#L90) instead of using `pip-sync-faster`. But our non-cookiecuttered apps have all been using `pip-sync-faster` for ages now. I fixed the cookiecutter's `tox.ini` to use `pip-sync-faster` which also requires `pip-sync-faster` to be in these requirements files